requirements.txt

```diff
@@ -4,3 +4,4 @@ xarray
 cftime
 netCDF4
 dask
+psutil
```

Comment:
If this is only used in the tests, I don't think it should be added here, but rather in `ci/environment-dev-3.7.yml`.

Reply:
Good catch, thanks !

Reply:
Finally, no, the CI stuff seems to be not used. It's just github actions with the base dependencies.